src/Sql/dbo/Stored Procedures/Collection_CreateOrUpdateAccessForMany.sql

```diff
@@ -0,0 +1,95 @@
+CREATE PROCEDURE [dbo].[Collection_CreateOrUpdateAccessForMany]
```

Comment:
This looks good. Maybe some more comments would make it easier to maintain in future.